rclcpp/include/rclcpp/parameter.hpp

```diff
@@ -193,11 +219,53 @@ class ParameterVariant
     return parameter;
   }
 
+  std::string to_string() const
+  {
+    std::stringstream bytes;
```

Comment:
Since the stringstream is only used in one of the cases the variable should be moved to that scope.